config/rbac/role.yaml

```diff
@@ -6,6 +6,18 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
These permessions are for the operator controller, not for the metallb controller, right? If so, the commit message is misleading I think.

Reply:
right, fixing.